public/src/js/troubleshoot/views/stale.js

```diff
@@ -134,17 +134,18 @@ function fetchLastPressed (front) {
 }
 
 function checkPressedState (front, config, container, lastPress) {
+    const priority = config.fronts[front].priority;
```

Comment:
Maybe something like this?
```suggestion
    const priority = config.fronts[front].priority ? config.fronts[front].priority : "editorial";
```

Reply:
Interesting, it does seem that editorial is the default priority.